sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java

```diff
@@ -217,32 +218,40 @@ static RecognizeEntitiesResult getExpectedBatchCategorizedEntities2() {
      * @return A {@link RecognizeLinkedEntitiesResultCollection}.
      */
     static RecognizeLinkedEntitiesResultCollection getExpectedBatchLinkedEntities() {
-        LinkedEntityMatch linkedEntityMatch1 = new LinkedEntityMatch("Seattle", 0.0);
-        LinkedEntityMatch linkedEntityMatch2 = new LinkedEntityMatch("Microsoft", 0.0);
+        final TextDocumentBatchStatistics textDocumentBatchStatistics = new TextDocumentBatchStatistics(2, 2, 0, 2);
+        final List<RecognizeLinkedEntitiesResult> recognizeLinkedEntitiesResultList =
+            Arrays.asList(
+                new RecognizeLinkedEntitiesResult(
+                    "0", new TextDocumentStatistics(44, 1), null,
+                    new LinkedEntityCollection(new IterableStream<>(getLinkedEntitiesList1()), null)),
+                new RecognizeLinkedEntitiesResult(
+                    "1", new TextDocumentStatistics(20, 1), null,
+                    new LinkedEntityCollection(new IterableStream<>(getLinkedEntitiesList2()), null)));
+        return new RecognizeLinkedEntitiesResultCollection(recognizeLinkedEntitiesResultList, DEFAULT_MODEL_VERSION, textDocumentBatchStatistics);
+    }
 
-        LinkedEntity linkedEntity1 = new LinkedEntity(
-            "Seattle", new IterableStream<>(Collections.singletonList(linkedEntityMatch1)),
+    /**
+     * Helper method to get the expected linked Entities List 1
+     */
+    static List<LinkedEntity> getLinkedEntitiesList1() {
```

Comment:
my concern with having all this tests to look at specific values is that if the service changes something about their model, then our test will fail.
Make sure whenever u call this that the model-version is set to avoid it

Reply:
Sounds good. Will take care of model-version in another PR. Since TA hasn't apply the model version in other APIs as well. 

issue created: #13943